src/articles/articles.service.ts

```diff
@@ -11,16 +11,41 @@ export class ArticlesService {
     return this.prisma.article.create({ data: createArticleDto });
   }
 
-  findAll() {
-    return this.prisma.article.findMany({ where: { published: true } });
+  findAll(skip?: number, take?: number) {
+    if (skip && take) {
```

Comment:
Create query object instead of doing if else and pass the query to findAll method